next-app/src/routes/password/change/+page.svelte

```diff
@@ -16,6 +17,15 @@
 	$: if (form?.failed) {
 		$error = Error(form.failed)
 	}
+
+	$: new_password_confirm && debounce(() => verify(new_password_confirm, new_password), 400)
```

Comment:
I find this 400ms delay a bit quirky. It also puts the error message out of sync with whether the button is enabled or disabled. Considering the comparison check doesn't cost us anything, I think it make sense to reduce it or even just remove it.

Reply:
I also don't like getting error messages as I type the confirmation password 🤔. What do you think about always leaving the Submit button enabled? Can we only call verify on blur and submit instead?

Reply:
yeah, based on our recent conversation, I'll just get rid of the bells and whistles here and stick with an error once the user submits (and leave the button enabled the whole time)